run-ab-platform.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+VERSION=0.44.1
```

Comment:
I think we want this injected down on line 44 too:

```
base_github_url="https://raw.githubusercontent.com/airbytehq/airbyte-platform/v$VERSION/"
```

Otherwise this script will download `docker-compose.yaml`, `.env` and co from `main` rather than from the latest release tag. `main` can contain breaking changes in `docker-compose.yaml` (as a recent PR of mine did) and then people were running the new `docker-compose.yaml` with "old" images from the most recent release, which caused problems.

Reply:
Done and done!